QUICKSTART.md

```diff
@@ -1,10 +1,29 @@
 # Quickstart
 
-Let's walk through two quick examples to see how to start working with the e2e-testing framework. We have two similar use cases:
+Let's walk through two quick examples to see how to start working with the e2e-testing framework. But first, we need to understand how the tests work, and what dependencies we need to run them.
+
+- dependencies
```

Comment:
This is a bit odd. Was this perhaps a placeholder that you forgot to remove?

Reply:
Let me rephrase it:

```
First, we need to understand how the tests work, and what dependencies we need to run them.

- dependencies
- how do the tests work?
```
we add those two sections. And after that:

```
## Adding a new test suite

Let's walk through two quick examples to see how to start working with the e2e-testing framework.
```

Reply:
@cachedout I changed it in https://github.com/elastic/e2e-testing/pull/2838/commits/5d6668cb85e804b3a24d63850df66034d791ee95